styles/popup.css

```diff
@@ -0,0 +1,198 @@
+@media screen and (max-width: 992px) {
+  .popup-content {
+    display: none;
+  }
+
+  .popup-body {
+    background: #fff;
+    position: fixed;
+    top: 5%;
+    left: 2.5%;
+    overflow-y: scroll;
+    border: 1px solid #dfe1e6;
+    border-radius: 16px;
+    margin: 0 20px 40px 0;
+    padding: 16px;
+    height: 90vh;
+  }
+
+  .popup-btns {
+    display: flex;
+    font-family: "Poppins", sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 16px;
+    letter-spacing: 0.03em;
+    color: #6070ff;
+    list-style: none;
+    padding-left: 0;
+    margin: 24px 0 42px 0;
+  }
+
+  .view-popup p {
+    display: flex;
+    padding: 10px;
+    border: solid black;
+    border-radius: 8px;
+    border-width: thin;
+    border-color: #6070ff;
+  }
+
+  .popup-btns li p {
+    margin-right: 32px;
+    text-decoration: none;
+    background-color: #ebebff;
+    border-radius: 8px;
+    padding: 4px 12px;
+    margin-top: auto;
+  }
+
+  .popup-section {
+    padding: 18px 3%;
+  }
+
+  .popup-buttons {
+    display: flex;
+    gap: 10px;
+    border-top: 1px solid #e5e5e5;
+    padding: 30px 0 15px 0;
+    margin-left: 10px;
+  }
```

Comment:
- [ ] Please try to match the Figma design, currently, your popup buttons from the desktop does not have a border, Add a `boder` property to the `See live` and `See Source button` :point_down: 

- [OPTIONAL ] Also, please consider displaying more than three technologies when a user visit the desktop version as you can see to the Figma design

Yours  | Figma
------------- | -------------
![b11](https://user-images.githubusercontent.com/57562869/152518905-83a08480-6d34-4d63-9974-5b813a3d31a9.png) | ![g11](https://user-images.githubusercontent.com/57562869/152519011-757e1d01-9419-4771-acc5-9e3605eb6bb2.png)